.storybook/main.js

```diff
@@ -9,10 +9,11 @@ const path = require('path');
 const webpack = require('webpack');
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 const TS_CONFIG_PATH = path.resolve(__dirname, '../tsconfig.json');
+const docs = process.env.BROWSER ? process.env.BROWSER.toUpperCase() !== 'IE11' : true;
```

Comment:
I'm open to feedback on this env var name/value. What makes more sense?

Reply:
`BROWSER` makes sense for me. Is it meant to be used like `BROWSER=ie11 yarn start`?

Reply:
yep, as described ☝️ 😉